Add formatter status item (#549)
Shopify/vscode-ruby-lsp: README.md

);
```

## Formatting

When `rubyLsp.formatter` is set to `auto`, the extension will try to determine which formatter to use.

> If the bundle has a **direct** dependency on `rubocop` or `syntax_tree`, that will be used.

Reviewer: ```suggestion
If the bundle has a **direct** dependency on a formatter, such as `rubocop` or `syntax_tree`, that will be used.
```

> Otherwise, formatting will be disabled and you will need explicity choose one.

Reviewer: ```suggestion
Otherwise, formatting will be disabled and you will need add one to the bundle.
```
